src/vue/modules/assets/shared/components/balance-viewer.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div
+    class="balance-viewer__balances-wrp--direction-column"
+  >
+    <span class="balance-viewer__balances-label">
+      {{ 'assets.balances' | globalize }}
+    </span>
+    <span
+      v-for="(item, index) in baseAssets"
+      :key="index"
+    >
+      {{ { value: item.balance, currency: item.asset } | formatMoney }}
+    </span>
+    <router-link
+      class="balance-viewer__balances-link"
+      :to="vueRoutes.balances"
+      @click.native="showMoreBalances"
+    >
+      {{ 'assets.show-more' | globalize }}
```

Comment:
not a part of `assets` scope, actually

Reply:
@max-nes Do you mean what `balance-viewer.vue` is in the incorrect folder?